### PR TITLE
docs: IFS-4209 correct references for removed glossary entrys

### PR DESCRIPTION
**Pull request by msg-jstren**
### **PR Type**
Documentation

___

### **Description**
- Updated multiple references from glossary entries to referenzarchitektur documents across various modules.
- Simplified and clarified text regarding library usage and application components.
- Enhanced documentation types and usage guidelines for better clarity.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><details><summary>13 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Update reference for 'Anwendungskern' in exception core documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-exception-core/pages/nutzungsvorgaben/inhalt.adoc

<li>Updated reference from glossary to referenzarchitektur.<br> <li> Changed link for 'Anwendungskern'.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-f5dbea1df88a741b3803e93d981f09ede9bff712b265e452e238c81f4d049b9c">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Simplify library usage text in logging documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-logging/pages/nutzungsvorgaben/inhalt.adoc

<li>Removed glossary reference for 'Bibliotheken'.<br> <li> Simplified text regarding library usage.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-51657b0f988679efab4638b4bc7accc9ce78b1dcff3ba0ed68fb5fedda4e481d">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>umsetzung-persistenzschicht.adoc</strong><dd><code>Update reference for 'Anwendungskern' in persistence documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-persistence/pages/nutzungsvorgaben/umsetzung-persistenzschicht.adoc

<li>Updated reference from glossary to referenzarchitektur.<br> <li> Changed link for 'Anwendungskern'.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-d0a94a4f60f085ec7522ea026bac6f3f0c39e4ee6c5ddf069dcefb15f25c143c">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>thisdoc.adoc</strong><dd><code>Simplify library usage text in polling documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-polling/pages/nutzungsvorgaben/thisdoc.adoc

<li>Removed glossary reference for 'Anwendungslandschaft'.<br> <li> Simplified text regarding library usage.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-e7fec11c3f58ca195cb31a9123cf766399c54fd59cbde0f9d78c7267d754e7b3">+3/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>konzept.adoc</strong><dd><code>Update reference for 'Anwendungskomponente' in task documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-task/pages/konzept.adoc

<li>Updated reference from glossary to referenzarchitektur.<br> <li> Changed link for 'Anwendungskomponente'.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-fc0565843fe641b4460f9acdd06aa9d30c9c7540d53f7d7b1b5aae08442d7fbd">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Update reference for 'Anwendungskern' in monitoring documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-ueberwachung/pages/nutzungsvorgaben/inhalt.adoc

<li>Updated reference from glossary to referenzarchitektur.<br> <li> Changed link for 'Anwendungskern'.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-4918946b2fb519b6011d57f908ae8cff1b24c6f668ac7064870c51f9bcb8f3a2">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Simplify library usage text in utility documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-util/pages/nutzungsvorgaben/inhalt.adoc

<li>Removed glossary reference for 'Bibliothek'.<br> <li> Simplified text regarding library usage.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-a39bea02ee2df54ecc6f110f61a8292c027de6a99995c028704a35cdf1b04aa4">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>thisdoc.adoc</strong><dd><code>Simplify library usage text in utility introduction</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-util/pages/nutzungsvorgaben/thisdoc.adoc

<li>Removed glossary reference for 'Bibliothek'.<br> <li> Simplified text regarding library usage.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-d56a1cbf385434e45839c3576d0526fb1d30885c12e0b4712292795d913e17dc">+3/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>dokumentationstypen.adoc</strong><dd><code>Enhance documentation types and usage guidelines</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/methodik/pages/vorgaben-dokumentation/dokumentationstypen.adoc

<li>Added details to technical documentation types.<br> <li> Clarified usage guidelines.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-23786a0587cd58114cdc213883e581e4da5d4f800ebe6493f2b78c25b3bc19fa">+4/-4</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>fachlich.adoc</strong><dd><code>Simplify application component text in architecture documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/fachlich.adoc

<li>Removed glossary reference for 'Anwendungsdomäne'.<br> <li> Simplified text regarding application components.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-352f90d518ee4d480c799463c063b2c097c5fa736c10446f1c09fd40ba036883">+2/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>anwendungskern.adoc</strong><dd><code>Simplify application core text in backend documentation</code>&nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/anwendungskern.adoc

<li>Removed glossary reference for 'Anwendungskern'.<br> <li> Simplified text regarding application core.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-7472fa65ed3427369e248c09109d3546b1ef414971fa46f873a56deb8c2d7ea2">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>services.adoc</strong><dd><code>Update reference for 'Anwendungskern' in services documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/services.adoc

<li>Updated reference from glossary to referenzarchitektur.<br> <li> Changed link for 'Anwendungskern'.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-513ca24a1181cc0a3c165df4954d51e3fc53955c9e9e1a543a03b16ccd536e46">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>betriebliche-aspekte.adoc</strong><dd><code>Simplify delivery text in operational aspects documentation</code></dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/technisch-infrastrukturell/betriebliche-aspekte.adoc

<li>Removed glossary reference for 'Auslieferung'.<br> <li> Simplified text regarding delivery.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/614/files#diff-26e94e5822be7e3f2a90744358cdc6731cac350f4e70cf50dd5d703885da7dfe">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></details></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # 4.0.0
 ## FEATURES
+- `IFS-4209`: [isyfact-standards-doc] Anpassung einiger Glossar-Referenzen aufgrund der Entfernung von Glossareinträgen in `isy-documentation`
```
Comment (stdll) [inline CHANGELOG.md]:
```suggestion
- `IFS-4209`: [isyfact-standards-doc] Anpassung einiger Glossar-Referenzen aufgrund der Entfernung von Glossareinträgen in [`isy-documentation`](https://github.com/IsyFact/isy-documentation).
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/fachlich.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ image::referenzarchitektur:fachlich/fachliche-referenzarchitektur-anwendungsland
 
 Zur Strukturierung der xref:glossary:glossary:master.adoc#glossar-anwendungslandschaft[Anwendungslandschaft] werden drei fachliche Hierarchieebenen festgelegt:
 
-*xref:glossary:glossary:master.adoc#glossar-anwendungsdomaene[Anwendungsdomäne]:*
+*xref:glossary:glossary:master.adoc#glossar-domaene[Anwendungsdomäne]:*
```
Comment (stdll) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/fachlich.adoc]:
```suggestion
*xref:glossary:glossary:master.adoc#glossar-domaene[Domäne]:*
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/technisch-infrastrukturell/betriebliche-aspekte.adoc [hunk 3]
```diff
@@ -74,15 +74,15 @@ Gleiches gilt, wenn sich zum Beispiel die Parameter eines Anwendungsservers änd
 Eine neue Version der Parameter liegt vor und damit liegt auch eine neue Konfiguration der Betriebsumgebung vor.
 Um nachvollziehen zu können, was sich wann warum geändert hat, ist es empfehlenswert, dass jede Veränderung an der Betriebsumgebung genehmigt und nachvollziehbar dokumentiert wird.
 
-*xref:glossary:glossary:master.adoc#glossar-auslieferung[Auslieferung]:* Mit dem Begriff „Auslieferung“ wird die Übergabe von Liefergegenständen aus der Hoheit der Software-Entwicklung in die Hoheit des Betriebs bezeichnet.
+*Auslieferung:* Mit dem Begriff „Auslieferung“ wird die Übergabe von Liefergegenständen aus der Hoheit der Software-Entwicklung in die Hoheit des Betriebs bezeichnet.
```
Comment (stdll) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/technisch-infrastrukturell/betriebliche-aspekte.adoc]:
```suggestion
*Auslieferung:* Mit dem Begriff "Auslieferung" wird die Übergabe von Liefergegenständen aus der Hoheit der Software-Entwicklung in die Hoheit des Betriebs bezeichnet.
```

**Review by stdll — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (msg-jstren) [reply CHANGELOG.md]: Done

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/fachlich.adoc (hunk 2 above)
Comment (msg-jstren) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/fachlich.adoc]: Done

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/technisch-infrastrukturell/betriebliche-aspekte.adoc (hunk 3 above)
Comment (msg-jstren) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/technisch-infrastrukturell/betriebliche-aspekte.adoc]: Done

**Review by stdll — APPROVED** [review]

**Pull request merged**